.github/workflows/main-CI.yml

```diff
@@ -42,6 +42,7 @@ jobs:
         env:
           S3_ENCRYPT_KEY: ${{ secrets.S3_ENCRYPT_KEY }}
           DEV_SECRET: ${{ secrets.DEV_SECRET }}
+          GLOBAL_ENV_BUCKET: some-bucket
```

Comment:
This stems from tibanna_ff [here](https://github.com/4dn-dcic/tibanna_ff/blob/master/tibanna_ffcommon/vars.py#L17-L19). There are several variables like this that should probably be refactored out into separate modules to prevent such behavior, as we have to set dummy variables for magma's GA as well, but I didn't want to open that can of worms for the dependency update here.